### ISSUE TITLE
Non-ublue specific images

### ISSUE THREAD

**Issue by tulilirockz** [atimeofday/image-switcher#2]
Maybe there should be a way to run non `ublue-os` organization images. Currently the code is hard coded to `ghcr.io/ublue-os`, but eventually there should be a way to use other registries and repositories n whatever else.

**Comment by atimeofday** [comment]
Yes, good note. I intend for this to work with both ublue and upstream Fedora Silverblue/Kinoite/etc images, in both directions. I intend to add that logic but if someone beats me to it, that works too
